src/ray/raylet/scheduling_resources.cc

```diff
@@ -505,14 +686,44 @@ const ResourceSet &SchedulingResources::GetLoadResources() const {
 
 // Return specified resources back to SchedulingResources.
 bool SchedulingResources::Release(const ResourceSet &resources) {
-  return resources_available_.AddResourcesStrict(resources);
+  bool avail_status = resources_available_.AddResourcesCapacityConstrained(resources, resources_total_);
```

Comment:
avoid using abbreviations everywhere, so `avail` should be `available`.

Also in this case, it's not clear what `avail_status` means. Maybe choose a better name?